learn/tanssi/appchain-services/block-production.md

```diff
@@ -0,0 +1,182 @@
+
+Finally, Tanssi can verify that the author of an appchain block was the expected one and reward accordingly.
+
+The following diagram shows a simplified model of the data Tanssi stores in its internal state. For every active appchain (in this example, two), Tanssi stores the assigned sequencers, which are the only ones authorized to produce blocks on the appchain's behalf, proof of validity (candidate receipts) extended by the security provider's operators, the latest state root, and the latest block producer. 
+
+```mermaid
+flowchart TB
+
+    subgraph general["Orchestration Info"]
+        direction TB
+        assignment1["Appchains Current Sequencer Assignment"]
+        assignment2["Appchains Next Session's Sequencer Assignment"] 
+    end
+    subgraph appchain1["Appchain 1"]
+        direction LR
+        appchain11["Current Sequencers assigned"]
+        appchain12["Last Block's Proof of Validity"]
+        appchain13["Latest State Root"]
+        appchain14["Latest Block Producer"]
+    end
+    subgraph appchain2["Appchain 2"]
+        appchain21["Current Sequencers assigned"]
+        appchain22["Last Block's Proof of Validity"]
+        appchain23["Latest State Root"]
+        appchain24["Latest Block Producer"]
+    end
+
+    general --> appchain1
+    general --> appchain2
+
+```
+
+### The Role of the Appchain {: #appchain }
+
+As a sequencer assigned to a Tanssi appchain includes built-in Tanssi node functionality, it is technically feasible to read the state from the Tanssi network.
+
+Leveraging this ability to access the states, the current sequencer with the authority to produce a block will read the state of the latest block produced in the Tanssi chain. It will proceed to include this state in the block of the appchain, the current set of sequencers assigned to the appchain, and its public signature, allowing Tanssi to know who produced the block and reward the node operator.
+
+Once the block is filled with appchain transactions, it will be proposed as a candidate and handed over to the Tanssi chain, where the security provider's operators will ensure that the included state proofs match the state proofs from the latest state of Tanssi (preventing unauthorized block production) and that the transactions produced valid state transitions. Having verified the work of the sequencer, the operators will finalize the proposed block, including its candidate receipt in a Tanssi network block.
+
+```mermaid
+classDiagram
+
+    note for tanssi "The sequencer reads the Tanssi State"
+
+    class tanssi["Appchain Block"] {
+        Appchain's Current Sequencers Assignment
+        Tanssi's State Root
+        Sequencer operator's signature
+        Appchain Transactions()
+    }
```

Comment:
Done, but It will not be valid for polkadot, since architecturally speaking both security approaches are different.
In the future we could include sections for how it works on each security provider